_partials/_migrate_dual_write_backfill_getting_help.md

```diff
@@ -0,0 +1,15 @@
+import OpenSupportRequest from "versionContent/_partials/_migrate_open_support_request.mdx"
+
+<Highlight type="tip">
+
+The dual-write and backfill migration method is relatively new, so the
```

Comment:
Maybe too discouraging for users to try? We can skip this sentence or something neutral like "In this doc we outline the dual-write and backfill migration method. In case you get stuck on any step..."

Reply:
done